packages/base/src/mainview/mainView.tsx

```diff
@@ -647,7 +647,9 @@ export class MainView extends React.Component<IProps, IStates> {
       color:
         layerParameters.type === 'fill' || layerParameters.type === 'circle'
           ? layerParameters.color
-          : '#F092DD'
+          : layerParameters.type === 'line'
+            ? 'rgba(0, 0, 0, 0)'
+            : '#F092DD'
```

Comment:
Should we really keep that #F092DD default? 

I'm not sure I understand in which case this #F092DD value will be used

Reply:
> Should we really keep that #F092DD default?
> 
> I'm not sure I understand in which case this #F092DD value will be used

I kept it for just in case 😅. However the above conditional logic covers all existing types. I'm happy to remove it.

Reply:
Done.